includes/class-social.php

```diff
@@ -147,7 +147,7 @@ public function widget( $args, $instance ) {
 
 		foreach ( $fields as $field ) {
 
-			if ( empty( $field['value'] ) || ! $field['show_front_end'] ) {
+			if ( empty( $field['value'] ) || ! $field['show_front_end'] || $this->is_social_media_deprecated( $field ) ) {
```

Comment:
@EvanHerman I don't think we have to make any changes to the front-end widget output. If a field has a value, we should display it.